conductor_api/src/conductor/base.rs

```diff
@@ -1603,6 +1612,37 @@ pub mod tests {
         )
     }
 
+    #[test]
```

Comment:
@maackle Can you confirm this test covers the required use case?

Reply:
nice test! :+1: